Permanent/Common/Extensions/ColorExtension.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  ColorExtension.swift
+//  Permanent
+//
+//  Created by Lucian Cerbu on 25.10.2023.
+
+import SwiftUI
+
+public extension Color {
```

Comment:
We do have a Color extension already in the Colors file. Either move this there or create a separate file that contains  all extensions for Color.